ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -117,25 +125,29 @@
   public static void main(String args[]) throws Exception {
     OptionParser parser = new OptionParser();
 
-    ArgumentAcceptingOptionSpec<String> accountJsonFilePathOpt = parser.accepts("accountJsonPath",
+    ArgumentAcceptingOptionSpec<String> accountJsonPathOpt = parser.accepts("accountJsonPath",
         "The path to the account json file. The json file must be in the form of a json array, with each"
             + "entry to be an account in its json form.")
         .withRequiredArg()
-        .describedAs("account_json_file_path")
+        .describedAs("account_json_path")
         .ofType(String.class);
 
+    ArgumentAcceptingOptionSpec<String> accountsToEditOpt = parser.accepts("accountsToEdit",
+        "The names of the accounts to edit. "
+            + "The script will open an editor where you can edit the json for these accounts.")
+        .withRequiredArg()
+        .describedAs("account_json_file_path")
+        .withValuesSeparatedBy(",");
     ArgumentAcceptingOptionSpec<String> zkServerOpt =
         parser.accepts("zkServer", "The address of ZooKeeper server. This option is required.")
             .withRequiredArg()
             .describedAs("zk_server")
             .ofType(String.class);
 
     ArgumentAcceptingOptionSpec<String> storePathOpt = parser.accepts("storePath",
-        "The root path of helix property store in the ZooKeeper. Must start with /, and must not end "
-            + "with /. It is recommended to make root path in the form of /ambry/<clustername>/helixPropertyStore. This option is required.")
-        .withRequiredArg()
-        .describedAs("helix_store_path")
-        .ofType(String.class);
+        "The root path of helix property store in the ZooKeeper. Must start with /, and must not end with /. "
+            + "It is recommended to make root path in the form of /ambry/<clustername>/helixPropertyStore. "
+            + "This option is required.").withRequiredArg().describedAs("helix_store_path").ofType(String.class);
```

Comment:
It seems to have converged on this formatting (since the chain of calls all fit on one line), but I changed the splitting of the string to coerce them into being on different lines.